README.rst

```diff
@@ -132,6 +132,47 @@ When not running in development mode, an environment variable named
 is to use the code Django includes for this purpose:
 ``SECRET_KEY=$(python -c "from django.core.management import utils; print(utils.get_random_secret_key())")``.
 
+Using Nix
+~~~~~~~~~
+
+You can install VulnerableCode with `Nix <https://nixos.org/download.html>`__ (`Flake <https://nixos.wiki/wiki/Flakes>`__ support is needed).
+
+::
+
+    nix --print-build-logs flake check # build & run tests
+
+There are several options to use the Nix version
+
+::
+
+    # Enter an interactive environment with all dependencies setup.
+    nix develop
+    > ./manage.py ... # invoke the local checkout
+    > vulnerablecode-manage.py ... # invoke manage.py as installed in the nix store
+
+    # Test the import prodecure using the Nix version.
+    ./test-import-using-nix.sh --all # import everything
+    # Test the import using the local checkout.
+    INSTALL_DIR=. ./test-import-using-nix.sh ruby # import ruby only
+
+
+**Keeping the Nix setup in sync**
```

Comment:
Not sure why a poetry conversion hack would be necessary, when this project could be packaged with `buildPythonPackage`.